src/models/WidgetData.ts

```diff
@@ -11,41 +11,19 @@
 //     tags: Array<string>;  // zero or more per widget
 //   };
 // }
-
 import { IPoint, IPointAccident } from './Point';
-
+import { IWidgetDataType } from './WidgetDataType';
 export interface ILocationData {
   meta: ILocationMeta;
   widgets: IWidgetBase[];
 }
-
 export interface IWidgetBase {
   rank: number; // order? (not in use)
   name: string; // unique - used as identifier
   data: IWidgetDataType;
```

Comment:
* replace `IWidgetDataType` with a new  interface (cane be named  `IWidgetDataBase`)
* All interfaces of `IWidgetDataType` should inherit from `IWidgetDataBase`;

Reply:
Is `title` always valid?
if not - it should be optional:
```
export interface IWidgetDataBase {
  text: {
    title?: string;
  };
}
```